### DISCUSSION TITLE
Combo box Filter / The place holder text is not revelant.

### DISCUSSION THREAD

**Discussion by hervedorierai3** [microsoft-search/pnp-modern-search#3018] [Ideas]
**Version used**
v4

**Describe the bug**
A bit too much to call that a bug, but the issue has been identified by a UX/UI auditing concerning the place holder text of a filter combo box.

**To Reproduce**
1. Configure a filter with a combo box template (either multivalues or single value).

The filter should appears like this : 
![image](https://user-images.githubusercontent.com/68116737/121044302-b7b48080-c7b5-11eb-9fa8-ef9da6ce8018.png)
The place holder text is "Select value".

**Expected behavior**
In a UX perspective, the place holder text is wrong... the user can select a value, but he can search for a value too.
So, I think that a text like "Search or select value" should be better.

I suggest multiple options : 
- change the text in the localized strings (hard coded).
- add a parameter in the filter webpart settings  : the place holder text will be the same for all combo box filter.
- add a parameter in each filter settings of the filter webpart : the place holder text can be the customized for each combo box filter (Please, select a document type).

**Screenshots**
None

**Desktop (please complete the following information):**
 - Windows 10 / Edge Chromium.

**Comment by wobba** [comment]
Feel free to PR a text change for all supported languages :) And not sure "search" is a good term either.. as it's more of a "filtering" operation.
